tests/integration/IdnaConvertDecodeTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+namespace Algo26\IdnaConvert\test;
```

Comment:
consistency? use `Test` as namespace, not lowercase

Reply:
You're right!